sr_gui_joint_slider/src/sr_gui_joint_slider/sliders.py

```diff
@@ -55,23 +53,23 @@ class Joint():
     Contains the name, and a controllers list for the joint.
     """
 
-    def __init__(self, name, min, max, vel, joint_controller):
+    def __init__(self, name, min_value, max_value, vel, joint_controller):
         self.name = name
         self.controller = joint_controller
         # Here we manipulate the limits to be in degrees (if they have to do with angles)
         # And also use vel to set min and max if the controller for this joint
         # is a velocity controller
-        self.min = round(degrees(min), 1)
-        self.max = round(degrees(max), 1)
+        self.min_value = round(degrees(min_value), 1)
```

Comment:
Same `min_position` ?

Reply:
Correct me if I am wrong, but I don't think I can name this specifically min_position as this gets overridden with either 'velocity' or 'pwm' related values. Seems that they are min/max_values based for the currently selected controller.

Reply:
Yes, that's true